src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -41,34 +49,34 @@
       <!-- List Content -->
       <v-row
         class="people-roles-content section-container"
-        :class="{
-          'summary-view': isSummaryView,
-          'invalid-section': invalidOrgPersons
-        }"
+        :class="{ 'summary-view': isSummaryView, }"
         v-for="(orgPerson, index) in currentPeopleAndRoles"
         :key="index"
         no-gutters
       >
         <!-- conditionally render edit component instead of table row -->
-        <div id="people-roles-edit" v-if="renderOrgPersonForm && (index === activeIndex)">
-          <OrgPerson
-            :currentOrgPerson="currentOrgPerson"
-            :activeIndex="activeIndex"
-            :currentCompletingParty="currentCompletingParty"
-            @addEdit="emitAddEdit($event)"
-            @remove="emitRemove($event)"
-            @reset="emitReset()"
-            @removeCpRole="emitRemoveCpRole()"
-          />
-        </div>
-
-        <template v-else>
+        <v-expand-transition>
```

Comment:
I added this transition to soften the impact when the component inside is rendered or destroyed.